src/main/java/entity/Monster/Monster.java

```diff
@@ -0,0 +1,120 @@
+package entity.Monster;
+
+import entity.Character;
+
+import java.util.HashMap;
+
+import java.util.Random;
+
+
+/** A Monster that the Player can encounter in the game.*/
+public class Monster extends Character {
+
+    /** The name of the Monster. */
+    private final String NAME;
+
+    /** The type of the Monster. */
+    private final String TYPE; // Maybe not String (so can be of multiple type)
+
+    /** The base attack of the Monster. */
+    private final int BASE_ATTACK;
+
+    /** The base health of the Monster. */
+    private final int BASE_HEALTH;
+
+    /** Whether the Monster has a power. */
+    private final boolean HAS_POWER; // not final to allow for possibility of removing powers
```

Comment:
is it final or not final? comment and actual code conflict

Reply:
LMAO sorry it's final